frontend/src/Dashboard/CrossingListPage/CrossingListItem/CrossingListItem.js

```diff
@@ -119,67 +127,103 @@ class CrossingListItem extends React.Component {
+        </div>
+      ) : (
+        <div className="CrossingListItemFlexItem--spacer" />
+      )}
+        <div className="CrossingListItemFlexItem">
+          <div className="ControlLabel">Notes to the public</div>
+          <input className="NotesTextBox" type="text" value={this.state.notes} onChange={this.notesChanged}/>
+        </div>
+      </div>
+
+      
+      { (show.includes('duration') || show.includes('cancelSave')) && (
+
+      <div className="CrossingListItemFlexContainer">
+        <div className="CrossingListItemFlexItem--spacer" />
+        
+      {show.includes('duration') ? (
+        <div className="CrossingListItemFlexItem">
+          <div className="ControlLabelContainer">
+            <div className="ControlLabel">Duration</div>
+            <div className="required">{this.isDirty() ? "Required" : ""}</div>
           </div>
-          <div className={show.includes('duration') || show.includes('cancelSave') ? "CrossingListItemFlexContainer" : ""}>
-            <div className="flexitem" />
-            <div className="flexitem">
-              <div className={show.includes('duration') ? "" : "hidden"}>
-                <div className="ControlLabelContainer">
-                  <div className="ControlLabel">Duration</div>
-                  <div className="required">{this.isDirty() ? "Required" : ""}</div>
-                </div>
-                <Dropdown
-                  options={durations}
-                  selected={this.state.selectedDuration}
-                  onChange={this.durationChanged} />
-              </div>
-            </div>
-            <div className="flexitem">
-              <div className={show.includes('cancelSave') ? "" : "hidden"}>
-                <div className="flexcontainer">              
-                  <div className="CancelButton" onClick={this.cancelClicked}>Cancel</div>
-                  <div className="SaveButton" onClick={this.newStatusUpdate.bind(this)}>Save</div>
-                </div>
-              </div>
-            </div>
+          <Dropdown
+            options={durations}
+            selected={this.state.selectedDuration}
+            onChange={this.durationChanged} />
+        </div>
+      ) : (
+        <div className="CrossingListItemFlexItem--spacer" />
+      )}
+
+      {show.includes('cancelSave') ? (
+        <div className="CrossingListItemFlexItem">
+          <div className="ButtonContainer">              
+            <div className="CancelButton" onClick={this.cancelClicked}>Cancel</div>
+            <div className="SaveButton" onClick={this.newStatusUpdate.bind(this)}>Save</div>
           </div>
         </div>
+      ) : (
+        <div className="CrossingListItemFlexItem--spacer" />
+      )}
+
+      </div>
+      )}
+
       </div>
+    );
+
+    if(!this.props.cqClassName) {
+      return (
+        <ContainerQuery query={containerQuery}>
+        {(params) => (
+          <div className={classnames(params, "CrossingListItem")}>
```

Comment:
We talked about making a javascript utility that would concatenate the string `CrossingListItem` with `--lg` instead of stacking the classes. Maybe that is more work than necessary and premature to address. But maybe also worth throwing into the Github Issue backlog. Just making a note so that we're on the same page.